ccd-definition/CaseField.json

```diff
@@ -619,5 +619,30 @@
     "Label": "Case management order",
     "FieldType": "CaseManagementOrder",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "orderBasis",
```

Comment:
Should we change this to be called `orderBasisLabel`?

Reply:
done